src/Utilities/Utilities.jl

```diff
@@ -363,15 +363,34 @@ end
 
 # Finding URLs -- based partially on code from the main Julia repo in `base/methodshow.jl`.
 
+function in_cygwin()
+    # check if in cygwin, if so we need to translate cygwin paths to windows paths
+    if is_windows()
+        try
+            success(`cygpath -h`)
+        catch
+            return false
+        end
+    else
+        return false
+    end
+end
+
 function url(repo, file)
     file = abspath(file)
     remote = getremote(dirname(file))
     isempty(repo) && (repo = "https://github.com/$remote/tree/{commit}{path}")
     commit, root = cd(dirname(file)) do
-        readchomp(`git rev-parse HEAD`), readchomp(`git rev-parse --show-toplevel`)
+        toplevel = readchomp(`git rev-parse --show-toplevel`)
+        if in_cygwin()
```

Comment:
Ok, I just realized that Julia interprets this as `C:\home\morten\julia`, and not as the location of the Cygwin home (which in my case should be `C:\cygwin64\home\morten\julia`). Should we go for an issue in Base?